src/gui/folderman.cpp

```diff
@@ -462,7 +466,7 @@ void FolderMan::slotFolderSyncPaused( Folder *f, bool paused )
 void FolderMan::slotFolderCanSyncChanged()
 {
     Folder *f = qobject_cast<Folder*>(sender());
-    Q_ASSERT(f);
+    RASSERT(f, return);
```

Comment:
If this happens it's a programming error and we should crash, I'd keep the Q_ASSERT and let it crash on the next line.